src/main/java/homework_2/PyramidPrinter.java

```diff
@@ -0,0 +1,23 @@
+package homework_2;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public class PyramidPrinter {
+    public static void main(String[] args) {
+        System.out.println("How many levels should the pyramid be?");
+        try (BufferedReader reader = new BufferedReader(new InputStreamReader(System.in))) {
+            try {
+                int levels = Integer.parseInt(reader.readLine());
+                for (int i = 1; i <= levels; i++) {
+                    System.out.println("x".repeat(i));
```

Comment:
Does not comile. Did I tell everybody to use java 8?

Reply:
I've used another computer for this homework, Didn't expect that the java version settings don't pull from a git branch.